apps/api/src/campaign-application/campaign-application.service.ts

```diff
@@ -86,12 +86,66 @@ export class CampaignApplicationService {
     }
   }
 
-  findAll() {
-    return this.prisma.campaignApplication.findMany()
+  async findAll() {
+    try {
+      const campaignApplications = await this.prisma.campaignApplication.findMany()
+      return campaignApplications
+    } catch (error) {
+      Logger.error('Error in findAll():', error)
+      throw error
+    }
+  }
+
+  async findOne(id: string, isAdminFlag: boolean, person: Person) {
```

Comment:
A bit ugly but set the type of the person to be 
```js
person: Prisma.PersonGetPayload<{ include: { organizer: {select:{id:true}}}}>
```

instead of `Person` to prevent the linter complaint. Also make sure that organizer.id is optional(e.g. person.organizer?.id)